docs/experimental-machine-learning/experimental-detections.md

```diff
@@ -0,0 +1,23 @@
+# Experimental ML Jobs and Rules
+
+Once data is being enriched, there are some rules and ML jobs which can leverage the enriched fields. 
+The experimental rules and jobs will be staged separate from the model bundle under the [releases](https://github.com/elastic/detection-rules/releases) 
+as `ML-experimental-detections-YYYMMDD-N`. 
+
+These releases will be additive, in that rule updates will come out in new releases.
```

Comment:
phrases like "in that" suggest that this sentence isn't structured very well. It's also not clear what this means

Reply:
Updated.